### PR TITLE
Fix check duplicate users

### PR DESCRIPTION
**Pull request by bprize15**
Fix https://github.com/oncokb/oncokb-pipeline/issues/191

### CODE REVIEW

File: src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/ManagedUserVM.java [hunk 1]
```diff
@@ -23,6 +23,8 @@ public class ManagedUserVM extends UserDTO {
 
     private Boolean needsMskRocReview = true;
 
+    private Boolean requestApiAccess;
```
Comment (zhx828) [inline src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/ManagedUserVM.java:26]: These should not be here.

**Review by zhx828 — CHANGES_REQUESTED** [review]
@bprize15 I mixed the changes related to API change. Could you update?

File: src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/ManagedUserVM.java (hunk 1 above)
Comment (bprize15) [reply src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/ManagedUserVM.java:26]: Sorry thought I stashed that

File: src/test/java/org/mskcc/cbio/oncokb/service/UserServiceIT.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import org.mskcc.cbio.oncokb.service.mapper.UserMapper;
 
 import io.github.jhipster.security.RandomUtil;
+import liquibase.pro.packaged.iu;
```
Comment (zhx828) [inline src/test/java/org/mskcc/cbio/oncokb/service/UserServiceIT.java:15]: We don't use this.

**Review by zhx828 — APPROVED** [review]
Thanks

**Pull request merged**